src/styles/pages/_pagination.scss

```diff
@@ -0,0 +1,46 @@
+@import '../settings/colors';
+
+$W_PAGINATION_LINK_SPACING: 6px;
+
+@mixin w-pagination__links($COLOR, $ACTIVE: false) {
+  display: flex;
+  font-size: 20px;
```

Comment:
I was wondering about switching this to `1rem` to make it a bit smaller and help it scale with the rest of the type. wdyt?